services/src/test/java/org/keycloak/vault/KeystoreVaultProviderTest.java

```diff
@@ -0,0 +1,66 @@
+package org.keycloak.vault;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.keycloak.vault.SecretContains.secretContains;
+
+/**
+ * Tests for {@link FilesKeystoreVaultProvider}.
+ *
+ * @author Peter Zaoral
+ */
+public class KeystoreVaultProviderTest {
```

Comment:
This is a unit test. Would it make sense to also add a Quarkus dist test for it? WDYT @Pepo48 @pedroigor